style.css

```diff
@@ -505,6 +513,60 @@ hr {
   flex-wrap: wrap;
 }
 
-.footer p {
+.footer {
+  background-color: #464646;
+}
+
+.footer p,
+a {
   margin: 0 0.25em;
+  color: #ffffff;
+  font-family: Al Nile;
```

Comment:
This font family is non-standard and doesn't work. Raphael needs to change it.

Reply:
I used what I saw in Figma